libs/api/domains/license-service/src/lib/client/driving-license-client/drivingLicenseService.api.ts

```diff
@@ -481,8 +490,14 @@ export class GenericDrivingLicenseApi
         // noop
       }
 
+      // Is there a status code from the service?
+      const serviceErrorStatus = result.error.serviceError?.status
+
+      // Use status code, or http status code from serivce, or "0" for unknown
+      const status = serviceErrorStatus ?? (result.error.statusCode || 0)
+
       error = {
-        status: (result.error.statusCode || 0).toString(),
+        status: status.toString(),
         message: result.error.serviceError?.message ?? 'Unknown error',
```

Comment:
```suggestion
        message: result.error.serviceError?.message || 'Unknown error',
```

Use fallback error message for non-nully-yet-empty messges?